example/spec/helpers/testUtils.js

```diff
@@ -193,6 +193,34 @@ function isValidAsyncOperationId(asyncOperationId) {
   return /[\da-f]{8}(?:-[\da-f]{4}){3}-[\da-f]{12}/.test(asyncOperationId);
 }
 
+function greenConsoleLog() {
+  const Bright = '\u001B[1m';
+  const colorReset = '\u001B[0m';
+  const FgGreen = '\u001B[32m';
+  return `${FgGreen}${Bright}%s${colorReset}`;
+}
```

Comment:
Cool!

Reply:
@Jkovarik I need help implementing the switchable for CI logic. Thanks

Reply:
@vpnguye2  I'd be happy to.   The idea is to switch the behavior if a bamboo-run-specifc env var is in place.     It can either be something we set in set-bamboo-env-varaibles or more narrowly in wherever we're going to execute this.   

It'd probably also be ok to punt this concern until we're actually running it in CI, if we have a ticket to run this test (e.g. add it to ... deployment, ????) addressing this in that AC would probably be fine as well. 

Feel free to ping me in a DM/channel if you want to pair on it.